index.md

```diff
@@ -108,14 +108,14 @@ viewer = napari.view_image(data.astronaut(), rgb=True)
 
 ![image](./images/screenshot-add-image.png)
 
-To do the same thing inside a script call
+To do the same thing inside a script call (not the call to `napari.run()` at the end)
```

Comment:
We need to say *why* `napari.run()` is important here, and why things are different than in IPython / jupyter notebooks.

Reply:
I feel like there's a balance to be struck here between "just get me going already, this is just the intro page", and "help me to understand why the event loop is different when I'm in IPython vs running in a script".  I was planning on updating the docs at https://napari.org/guides/stable/event_loop.html ...  would it be enough to link to that and say "see here for more details?"